qemu/tests/cfg/nic_hotplug.cfg

```diff
@@ -94,3 +94,19 @@
         - used_netdev:
             used_sameid = yes
             devadd_match_string = "it's in use"
+        - with_iommu:
+            only virtio_net
+            only nic_virtio
+            netdev_extra_params_hotplug_nic1 = ",vhost=on,vhostforce=on"
+            nic_extra_params_hotplug_nic1 = "disable-legacy=on,disable-modern=off,iommu_platform=on,ats=on"
+            x86_64:
+                only q35
+            aarch64, x86_64:
+                nic_extra_params_hotplug_nic1 = "iommu_platform=on,ats=on"
+                no WinXP WinVista Win7 Win8 Win8.1 Win2003
```

Comment:
Some lines can move to `x86_64:` part.

Reply:
I have changed it, please help me review it again. Thanks.